sql_generators/glean_usage/templates/event_monitoring_aggregates_v1.query.sql

```diff
@@ -2,7 +2,8 @@
 {% for app in apps %}
 {% set outer_loop = loop -%}
 {% for dataset in app -%}
-{% if dataset['bq_dataset_family'] not in ["telemetry", "accounts_frontend", "accounts_backend"] %}
+{% if dataset['bq_dataset_family'] not in ["telemetry", "accounts_frontend", "accounts_backend"]
```

Comment:
Would it be straightforward to wrap the if around only the different parts for `accounts_frontend` and `accounts_backend`, or are there too many small differences?

Reply:
Yeah, there are a couple of differences. Having two separate queries seemed more readable in this case